### ISSUE TITLE
Outdated SILVA database

### ISSUE THREAD

**Issue by Piplopp** [ablab/quast#102]
Hello !

I was installing quast and noticed the following line during the install:
```
gunzip -c quast_libs/silva/SILVA_123_SSURef_Nr99_tax_silva.fasta.gz > quast_libs/silva/SILVA_123_SSURef_Nr99_tax_silva.fasta.unpacked \
```

Is there a way to update this database ? The current latest SILVA release is the 132.

Have a nice day :)

**Comment by alexeigurevich** [comment]
Thanks for noticing!
We will update the SILVA DB in the next Quast release. As the current workaround, you can download the newest SILVA yourself and provide it to Quast using `--blast-db` option (actually works with any DB). See more info in [FAQ Q12](http://quast.bioinf.spbau.ru/manual.html#faq_q12) in our manual.

**Comment by alexeigurevich** [comment]
UPDATEs on this issue: developers version (`master` branch of this repo) is updated to v.132.
Also, I've made further updates much easier, one needs just to set up the proper version number [here](https://github.com/ablab/quast/blob/master/quast_libs/search_references_meta.py#L40)).

For those, who wish to upgrade the current stable QUAST release (5.0.2) with this new functionality, see the attached [archive](https://github.com/ablab/quast/files/3771629/silva_update.zip). Unpack it and move the content (`search_references_meta.py` and `qutils.py`) under `<quast_installation_dir>/quast_libs/` (replace old files with the same names).

--
Also related notice. SILVA [announced](https://www.arb-silva.de/news/view/2019/10/24/silva-ssu-release-138-in-november/) a new update very soon (v. 138 in mid-November). We will hopefully switch to it in the upcoming QUAST release.

**Comment by alexeigurevich** [comment]
We updated the SILVA database to the latest version 138 (Dec 2019) in the master branch and in the just-released pre-5.1.0: https://github.com/ablab/quast/releases/tag/quast_5.1.0rc1 
So, it will be available in the stable 5.1.0 release, too (soon!).
